src/components/DynamicCalendar/index.js

```diff
@@ -304,12 +304,13 @@ class DynamicCalendar extends Component {
               eventTitleArray[i],
               eventSubtitleArray[i]
             )
+
             markedDatesArray.push(
               this.formatDate(new Date(eventStarttimeArray[i]), false)
             )
           }
           let dates = this.getDates(new Date(startTime), new Date(endTime))
-          for (let j = 1; j < dates.length - 1; ++j) {
+          for (let j = 1; j < dates.length; ++j) {
```

Comment:
I'm pretty sure the function `getDates` just returns an array of dates from `startTime` to `endTime`, so I don't think it's month-specific. This code is such spaghetti code, oy.
I think this code is fine, especially if you could see it solving the problem.

@nucks, why do you think it would cause problems for other months?